Eluant/LuaApi.cs

```diff
@@ -294,6 +294,14 @@ public static void luaL_getmetatable(IntPtr L, string name)
         public static extern int luaL_loadstring(IntPtr L, [MarshalAs(UnmanagedType.LPStr)] string s);
 
         [DllImport(LUA_DLL, CallingConvention=LUA_CALLING_CONVENTION)]
+        public static extern int luaL_loadbuffer(IntPtr L, [MarshalAs(UnmanagedType.LPStr)] string s, UIntPtr size, string n);
+
+        public static int luaL_loadbuffer(IntPtr L, string s, int size, string n)
```

Comment:
For native call wrappers I'd be inclined to omit the `size` parameter and use `s.Length` here instead of at the call site.